python/tvm/schedule.py

```diff
@@ -140,6 +140,145 @@ def vstore(self, begin, value):
         return _api_internal._BufferVStore(self, begin, value)
 
 
+@register_node
+class Layout(NodeBase):
```

Comment:
It is a little bit strange to put Layout under `schedule.py`

Reply:
I agree, perhaps put it under Tensor